test/mpi/configure.ac

```diff
@@ -99,11 +99,8 @@ AC_ARG_ENABLE(echo,
 AC_ARG_ENABLE(fortran,
 [  --enable-fortran=option - Control the level of Fortran support in the MPICH implementation.
         yes|all   - Enable all available Fortran implementations (F77, F90, F08)
-        f77       - Enable Fortran 77 support
-        f90       - Enable Fortran 90 support
-        f08       - Enable Fortran 2008 support
         no|none   - No Fortran support
-],,[enable_fortran=f77,f90])
+],,[enable_fortran=all])
```

Comment:
Thanks for the review! It is not dumb at all. How to match the test suite to the mpich build is tricky. The issue we were having is even though mpich is built with f08, but the tests is not enabled unless we hard code the option into Jenkins' job. We have been missing the test coverage because of this.

Later in the configure there is code to check whether the compiler actually has `2008` support:
https://github.com/pmodels/mpich/blob/26cc6a9ea874b9588dc181ac70bfe4161b57493f/test/mpi/configure.ac#L1411-L1413
So if the compiler is too "old" (e.g. gcc-8), the tests will be disabled later.